thread/src/components/AddContactForm.js

```diff
@@ -36,18 +46,19 @@ class AddContactForm extends Component {
 
     render() {
         return (
-            <Container style={{ backgroundColor: '#fff' }}>
+            <Container style={{ backgroundColor: '#8bc34a' }}>
                 <Content>
-                    <Form>
+                    <Form style={{padding:20, marginLeft: -20}}>
```

Comment:
Duplicate code again seen another PR, move the Form component to a separate file